### ISSUE TITLE
Rolling out rocket while new launchpad is building causes bugs

### ISSUE THREAD

**Issue by Teykn** [KSP-RO/KCT#16]
_Note, this issue has the same bugs as the "Deleting pad while build new launchpad causes bugs" issue post I made a while ago_

So, as it turns out, the game does not like rolling out rockets when a launchpad finishes building. When rolling out a rocket, my launchpad finished, which caused these issues:
1. Build time stuck at 100-1000x speed whilst at 1x time warp
2. Launchpad stuck at 00:00:00:00 time
3. All science research just stops

**Comment by siimav** [comment]
Did you schedule 2 different tonnage pads to be build at the same time? If yes, is the order of your pads similar to that: 
1) 60t
2) 350t
3) 150t
This means that you can do rollout to the 150t pad at the same time as 350t pad gets completed.

**Comment by Teykn** [comment]
No, it was only one pad being built, the unlimited ton launchpad.

Only have 2 launchpads fully built and functional: 350t and 700t.
